### DISCUSSION TITLE
Another controller

### DISCUSSION THREAD

**Discussion by rk125** [syssi/esphome-jbd-bms#16] [Show and tell]
Thank you Sebastian for awesome software! 
Works perfectly with my JBD-AP20S006-L20S-200A-B with 16S batteries with esp8266 via serial interface (one transistor level shifters with open drain). Battery controller from 2022-04-08, sw version 2.3 or 23 (what XiaoXiang app says).
No need for separate raspberry with jbdtool any more, direct serial via usb to serial was fine but did not work well for me with esplink.

**Comment by syssi** [comment]
Thanks for your feedback! I've added the model to the list of supported devices!

**Comment by nnevers** [comment]
My bms keep chipping out when the inverter is on

**Reply by syssi** [reply]
Could you describe the issue in detail? Please enable the `debug` mode of the `uart` component and provide some logs. Try to increase the `rx_timeout` step by step.
